abdulfatai360/banka
# #165470000 migrate account endpoints apis to database

#### What does this PR do?
Have the accounts post, patch, and delete endpoints moved to database

#### Description of Task to be completed?
- Have the data of the accounts endpoint APIs persist
- Have the integration tests pass

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/44/head:ch-migrate-accounts-165470000`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-migrate-accounts-165470000`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165470000](https://www.pivotaltracker.com/story/show/165470000)
